src/Constants.py

```diff
@@ -1,22 +1,22 @@
 from enum import Enum
 
 EXIT_PAYMENT_TYPE = "exit"
-
-CURRENT_TESTNET = 'GRANADANET'
+PROTOCOL_NAME = 'GRANADA'
```

Comment:
Question: I tried updating the protocol name to `HANGZHOU` but it seems that the [current_level](https://hangzhounet.smartpy.io/chains/main/blocks/head/header/) of this protocol is at 169123. Which is significant less then in granada [current_level](https://rpc.tzkt.io/granadanet/chains/main/blocks/head/header) with 765770. 

I am asking because then it fails the `test_calculatePhase0.py` test. Because the snapshot_level (266241) is bigger then the current_level (169123) in `HANGZHOU`. See this if check [here](https://github.com/tezos-reward-distributor-organization/tezos-reward-distributor/blob/master/src/rpc/rpc_reward_api.py#L482). How do we want to deal with that?

Reply:
We will make tests with mainnet in order to avoid that trouble